IntegrationTests/allocation-counter-tests-framework/template/AtomicCounter/Sources/AtomicCounter/include/atomic-counter.h

```diff
@@ -22,7 +22,7 @@ void inc_malloc_counter(void);
 void reset_malloc_counter(void);
 long read_malloc_counter(void);
 
-void add_malloc_bytes_counter(size_t v);
+void add_malloc_bytes_counter(intptr_t v);
 void reset_malloc_bytes_counter(void);
 long read_malloc_bytes_counter(void);
 
```

Comment:
I think that's what you meant in https://github.com/apple/swift-nio/pull/1116/files#r315248349 @weissi ?
Please sanity check, I'm not 100% sure that's right.

Reply:
yes, thanks!